### DISCUSSION TITLE
👋 Welcome!

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [CRobbins0867/ProdigyHacking--MobileHacking#11] [General]
# 👋 Welcome!
You may use Discussions to communicate with others and ask questions about this repository.

Rules:

Be respectful
Follow GitHub's Terms of Use
Do not spam

**Comment by KittyMasta** [comment]
I requested a hacked account so how will i know the user and pass?

**Comment by ShawnC32** [comment]
Imagine friend botting

**Comment by XKTE925** [comment]
This shii don’t work no more

**Reply by joeZwhwjz8** [reply]
> This shii don’t work no more

it don't? damnnn that's weird asf

**Comment by XKTE925** [comment]
Is saying your pfp kinda bad in a good way respectful?
